fem/transfer.cpp

```diff
@@ -1753,16 +1753,23 @@ void L2ProjectionGridTransfer::L2ProjectionH1Space::TDofsListByVDim(
 void L2ProjectionGridTransfer::L2ProjectionH1Space::LumpedMassInverse(
    Vector& ML_inv) const
 {
-   Vector ML_inv_full(fes_lor.GetVSize());
+#ifdef MFEM_USE_MPI
+   auto * fes = pfes_lor_scalar.get();
+#else
+   auto * fes = fes_lor_scalar.get();
+#endif
```

Comment:
Serial examples should be supported with the parallel build, so we may need some logic here to use `fes_lor_scalar` even when `MFEM_USE_MPI` is defined.